### DISCUSSION TITLE
Galaxy Brain Achievement Test

### DISCUSSION THREAD

**Discussion by thatwonguy** [Schweinepriester/achievement-testarea#57] [Q&A]
To recap regarding the galaxy brain: 
![image](https://github.com/Schweinepriester/github-profile-achievements/assets/78534460/d2605614-345d-4f83-81d0-15e4e7be7567)

To unlock the Galaxy Brain badge on GitHub, you need to answer at least 2 discussions. The discussions must be in a public repository, and you cannot mark your own answers to your own questions.

**Comment by matteofuso** [comment]
Exactly, in order to get the Galaxy Brain Badge you must

1. Get at least 2 marked answers in public discussion
2. You can't mark your own responses

You may need more than 2 responses to discussions
